### PR TITLE
My updated work

### PR DESCRIPTION
**Pull request by Tamannahasin123**
Please review

### CODE REVIEW

File: src/test/java/Class3/Homework2.java [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        driver.findElement(By.xpath("//input[contains(@name,'last')]")).sendKeys("Haaaa");
+
+        //Enter mobile number
+        driver.findElement(By.xpath("//*[text()='Mobile number or email']")).sendKeys("999 999 9999");
+
+        //Enter new password
+        driver.findElement(By.xpath("//*[text()='New password']")).sendKeys("abc@123");
+
+        //Click the "Sign Up" button
+        driver.findElement(By.xpath("//*[text()='Sign Up' and @xpath='1']")).click();
```
Comment (deepaktechnosoft) [inline src/test/java/Class3/Homework2.java]: never use "xpath attribute in xpath....as it is there by Chropath to indicate the line of DOM with entered xpath

File: src/test/java/Class4/Homework3.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+     * TC-4: Verify correct temp value is displayed based on temp-unit
+     *
+     * Steps:
+     * 1. Launch darksky.net
+     * 2. Change the temp unit
+     * 3. Verify the temp-value is converted correctly as the selected-unit
+     *
+     */
+
+
```
Comment (deepaktechnosoft) [inline src/test/java/Class4/Homework3.java:53]: No homework code

**Review by Tamannahasin123 — COMMENTED** [review]
Hi Deepak, 

commits 076b31356cf7f686497a595d025408c68cf72d87
Please advise.

Thanks
Tamanna

File: src/test/java/Class3/Homework2.java (hunk 1 above)
Comment (Tamannahasin123) [reply src/test/java/Class3/Homework2.java]: Fixed

File: src/test/java/Class4/Homework3.java (hunk 2 above)
Comment (Tamannahasin123) [reply src/test/java/Class4/Homework3.java:53]: 076b31356cf7f686497a595d025408c68cf72d87

**Review by deepaktechnosoft — COMMENTED** [review]
Thanks

**Pull request closed without merging**